### ISSUE TITLE
[1.16.5] Yung api warn when make a world

### ISSUE THREAD

**Issue by quentin452** [quentin452/1.7.10-Forge-renewed-v1#224]
```
[22:46:38] [Server thread/WARN] [yungsapi/]: WARNING: block Block{minecraft:iron_ore} added to BlockSelector exceeds max probabiltiy of 1!

**Comment by quentin452** [comment]
fixed by removing Abyss2 in V0.81 HOTFIX

**Comment by quentin452** [comment]
fixed by idk
